### ISSUE TITLE
Mobile: Like emoji doesn’t work 

### ISSUE THREAD

**Issue by omniwired** [TERITORI/teritori-dapp#1034]

**Comment by WaDadidou** [comment]
I didn't reproduce.
Adding emojis on social posts works : the emoji appears after adding it and after reloading page.
Same for Like/Dislike on video posts

**Comment by omniwired** [comment]
Yes @WaDadidou . That's why @sachin-into fixed this one on a PR.
Otherwise it was confusing for users
